### ISSUE TITLE
RSpec false positive: Can't call create on an existing resource

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#406]
### Descriptive summary

RSpec frequently fails tests with this false positive:
```
Failure/Error: document_id { create(:work).id }

Ldp::Conflict:
  Can't call create on an existing resource
./spec/factories/oembed_errors.rb:5:in `block (3 levels) in <top (required)>'
./spec/models/oembed_error_spec.rb:6:in `block (2 levels) in <top (required)>'
./spec/models/oembed_error_spec.rb:10:in `block (2 levels) in <top (required)>'
./spec/rails_helper.rb:86:in `block (3 levels) in <top (required)>'
./spec/rails_helper.rb:85:in `block (2 levels) in <top (required)>'
```

It's likely some sort of race condition, and might have to do with there being two work builders

### Expected behavior

Above test passes non-sporadically

**Comment by CGillen** [comment]
Josh mentioned trying to add `sequence(:id) { |n| "abc123#{n}" }` to the file_set factory. This might help and could be applied to other factories?

Or try adding `clean_repo` metadata to some tests: `RSpec.describe FileSet, clean_repo: true do`

**Comment by revgum** [comment]
fixed by #496 ..  also fixed in Hyrax 2.5 (https://github.com/samvera/hyrax/releases/tag/v2.5.0)

**Comment by straleyb** [comment]
Could be fixed by that or it could be something based on DatabaseCleaner and truncating the database
